### PR TITLE
Change domain names for blogs and email

### PR DESCRIPTION
**Pull request by Ojself**
Change domains for bloglinks and email
https://ticketsystem.schul-cloud.org/browse/SC-6239

# Description

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Review by janrenz — DISMISSED** [review]

File: src/components/templates/CommonTermsOfUse.vue [hunk 1]
```diff
@@ -52,7 +52,9 @@
 		ermöglichen.<br /><br />
 		Beim Aufruf der Website durch nicht-angemeldete Besucher*innen ist das Hasso
 		Plattner Institut (HPI), Prof.-Dr.-Helmert-Str. 2-3, 14482 Potsdam,
-		<a href="mailto:datenschutz@schul-cloud.org">datenschutz@schul-cloud.org</a>
+		<a href="mailto:datenschutz@hpi-schul-cloud.de"
+			>datenschutz@hpi-schul-cloud.de</a
+		>
```
Comment (mamutmk5) [inline src/components/templates/CommonTermsOfUse.vue:57]: Line breaks as planned?

File: src/themes/n21/components/templates/CommonTermsOfUse.vue [hunk 2]
```diff
@@ -53,7 +53,9 @@
 		<br /><br />
 		Beim Aufruf der Website durch nicht-angemeldete Besucher*innen ist das Hasso
 		Plattner Institut (HPI), Prof.-Dr.-Helmert-Str. 2-3, 14482 Potsdam,
-		<a href="mailto:datenschutz@schul-cloud.org">datenschutz@schul-cloud.org</a>
+		<a href="mailto:datenschutz@hpi-schul-cloud.de"
+			>datenschutz@hpi-schul-cloud.de</a
+		>
```
Comment (mamutmk5) [inline src/themes/n21/components/templates/CommonTermsOfUse.vue:58]: Line breaks as planned?

File: src/themes/open/templates/impressum.vue [hunk 3]
```diff
@@ -11,7 +11,9 @@
 			14482 Potsdam
 			<br />
 			Email:
-			<base-link href="info@schul-cloud.org">info@schul-cloud.org</base-link>
+			<base-link href="info@hpi-schul-cloud.de"
+				>info@hpi-schul-cloud.de</base-link
+			>
```
Comment (mamutmk5) [inline src/themes/open/templates/impressum.vue:16]: Line breaks as planned?

File: src/themes/thr/components/templates/impressum.vue [hunk 4]
```diff
@@ -10,7 +10,9 @@
 			14482 Potsdam
 			<br />
 			Email:
-			<base-link href="info@schul-cloud.org">info@schul-cloud.org</base-link>
+			<base-link href="info@hpi-schul-cloud.de"
+				>info@hpi-schul-cloud.de</base-link
+			>
```
Comment (mamutmk5) [inline src/themes/thr/components/templates/impressum.vue:15]: Line breaks as planned?

**Comment by Ojself** [comment]
@schul-cloud-bot update screenshots

**Review by mamutmk5 — CHANGES_REQUESTED** [review]
Pleas change the version at the files _package-lock.json_ and _package.json_

File: src/themes/thr/components/templates/impressum.vue (hunk 4 above)
Comment (Ojself) [reply src/themes/thr/components/templates/impressum.vue:15]: no. linter did this. I will change it now

**Review by janrenz — APPROVED** [review]
line breaks will be fixed later

**Review by mamutmk5 — APPROVED** [review]

**Pull request merged**